include/osquery/core.h

```diff
@@ -92,10 +92,47 @@ osquery::QueryData query(const std::string& q, int& error_return, sqlite3* db);
  * virtual tables attached. This method is the main abstraction for creating
  * SQLite3 databases within osquery.
  *
+ * Note: osquery::initOsquery must be called before calling createDB in order
+ * for virtual tables to be registered.
+ *
  * @return a SQLite3 database with all virtual tables attached
  */
 sqlite3* createDB();
 
+
+/**
```

Comment:
There is a `include/osquery/sql.h` in the public API: https://github.com/facebook/osquery/blob/master/include/osquery/sql.h

`sql.h` might be a better place, or `tables.h` what do you think?